blog/Getting Started with ANP.md

```diff
@@ -0,0 +1,274 @@
+# Getting started with the AdminNetworkPolicy API
+
+Let's first consider a couple of very typical networking requirements for cluster admins:
+
+1. There are certain traffic patterns in the cluster that, as a cluster admin, I want to
+   always allow. For example, all namespaces should allow ingress traffic from a monitoring
+   application in the cluster for compliance reasons, and all namespaces should allow egress
+   toward kube-dns running in the kube-system namespace.
+
+2. As a cluster admin I want to change the default security model for my cluster, so that
+   all inter-tenant traffic (traffic between namespaces with different tenant labels) is
+   blocked by default, unless it is known traffic explicitly allowed by namespace owners.
+
+Both of the intents above however, are extremely difficult, if not impossible, to be
+implemented with NetworkPolicies. For example, often developers accidentally block required
+traffic (most commonly dns) when they apply NetworkPolicies. This is due to the implicit
+deny nature of the NetworkPolicy semantics: pods selected by NetworkPolicies are automatically
+isolated except for any ingress/egress traffic that is explicitly allowed.
+So, without any admin-facing policy APIs, to "ensure" that dns always works and monitoring
```

Comment:
The reader may benefit from section headers

Seems like we have:
- Problem Overview
- NetPolV1 Solution and Limitations
- Improved Solution with Admin policies
- Connectivity example with AdminNetworkPolicy
- Continued example with BaselineAdminNetworkPolicy
- Takeaways & Next Steps